client/src/api.ts

```diff
@@ -34,6 +39,7 @@ export const startNewGame = async (numberPlayer: number) => {
 
 export const pullNewGame = async (playerNumber: number): Promise<GameState> => {
   try {
+    console.log(`URL :`, URL);
```

Comment:
debug spotted

Reply:
These aren't the Droids you're looking for...
removed